### ISSUE TITLE
replaceIconsForResource sets the wrong height for icons with an alpha mask

### ISSUE THREAD

**Issue by AlexanderOMara** [jet2jet/resedit-js#4]
Icons with an alpha mask have their heights doubled, which leads to `replaceIconsForResource` setting the wrong height values, and software which displays those icons choosing the wrong icons to display.

[I've created a Gist to demo the issue.](https://gist.github.com/AlexanderOMara/ce16b3ab90d5373857d48fba7bf3de63)

Essentially `RawIconItem` and `IconItem` may not have the correct `height` value for an icon group entry (possibly doubled for the alpha mask), but the array entries returned from `resedit.Data.IconFile.from` do. However only the data property of those entries (`RawIconItem` and `IconItem`) are passed to `replaceIconsForResource` so the wrong values are written to the icon group.

So if I use icons with sizes like this:
```
icons:
  ico: 0 0
  raw: 256 256

  ico: 128 128
  bmp: 128 256

  ico: 32 32
  bmp: 32 64
```

I get a group with sizes like this:
```
group:
  icon: 0 0
  icon: 128 0
  icon: 32 64
```

Instead of the correct values:
```
group:
  icon: 0 0
  icon: 128 128
  icon: 32 32
```

**Comment by jet2jet** [comment]
It seems that `IconItem` does not use the original width/height values. In case loading icons from the file, width/height values specified in the file should be used.

I'll fix this.

**Comment by jet2jet** [comment]
I fixed with #5.
If problem still occurs, please reopen the issue.

Thanks!
